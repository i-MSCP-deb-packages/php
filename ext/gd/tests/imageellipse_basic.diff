001+ d46b85bac6927352e9abc82e2a346286
001- d8b9bc2ca224bd68569413f4617f8e1f