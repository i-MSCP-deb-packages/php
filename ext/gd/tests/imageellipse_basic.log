
---- EXPECTED OUTPUT
d8b9bc2ca224bd68569413f4617f8e1f
---- ACTUAL OUTPUT
d46b85bac6927352e9abc82e2a346286
---- FAILED
