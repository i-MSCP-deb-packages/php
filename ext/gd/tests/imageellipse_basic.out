d46b85bac6927352e9abc82e2a346286